astropy/convolution/tests/test_pickle.py

```diff
@@ -1,8 +1,11 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
 import numpy as np
+
 from ... import convolution as conv
-from ...tests.helper import pytest, pickle_protocol, check_pickling_recovery
+from ...tests.helper import pickle_protocol, check_pickling_recovery  # noqa
```

Comment:
Because pyflakes doesn't recognize the use of pickle_protocol as a function argument to be a real use of the variable, so it thinks pickle_protocol is never used.